packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -13,12 +14,132 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
       backgroundColor: t.colors.transparentBackground,
       color: t.colors.brandForeground1,
       borderColor: t.colors.brandForeground1,
-      iconColor: t.colors.brandedBorder,
+      iconColor: t.colors.brandForeground1,
     },
     tint: {
-      backgroundColor: t.colors.brandedBackground,
-      color: t.colors.brandForeground2,
-      borderColor: t.colors.brandForeground2,
-      iconColor: t.colors.brandStroke2,
+      backgroundColor: globalTokens.color.brand.tint60,
```

Comment:
Is this a duplicate of the brand tokens?

Reply:
Also the issue with hooking into color.brand directly here is that it won't update to be consistent with the host's theme. tint60 would be closest to brandBackground2, so I think it would be better to use that here.

Reply:
Yeah, I added to PR description that theme support will be later because I had some questions about colors